tests/Aspire.EndToEnd.Tests/Directory.Build.props

```diff
@@ -4,5 +4,9 @@
     <IsTemplateTestProject>true</IsTemplateTestProject>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(BaseOutputPath)' == '' and '$(TestsRunningOutsideOfRepo)' == 'true' and '$(PrepareForHelix)' != 'true' ">
+    <BaseOutputPath >$([MSBuild]::NormalizeDirectory($([System.IO.Path]::GetTempPath()), $([System.IO.Path]::GetRandomFileName())))</BaseOutputPath>
+  </PropertyGroup>
```

Comment:
This is how we get the E2E project to run in the "out of repo" mode. It also allows us testing it locally with ease.